src/main.ts

```diff
@@ -54,9 +54,9 @@ async function start () {
     if (sentryDsn) {
       console.info('Initializing Sentry...') // eslint-disable-line no-console
       Sentry.init({
+        Vue,
         dsn: sentryDsn,
         integrations: [
-          new Integrations.Vue({ Vue, attachProps: true }),
           new Integrations.CaptureConsole({ levels: ['error'] })
```

Comment:
Is this still the right syntax to add this integration? Can you test that it works?

Reply:
I tested by adding:
  console.error('HELLO7')
  Sentry.captureException(new Error('HELLO8'))

Both make it to Sentry

Reply:
Excellent. Thanks!